deploy-service/common/src/main/java/com/pinterest/deployservice/handler/PingHandler.java

```diff
@@ -227,7 +229,7 @@ boolean canDeploy(EnvironBean envBean, String host, AgentBean agentBean) throws
         String envId = envBean.getEnv_id();
         AgentCountBean agentCountBean = agentCountDAO.get(envId);
         long totalNonFirstDeployAgents = (isAgentCountValid(envId, agentCountBean) == true) ? agentCountBean.getExisting_count() : agentDAO.countNonFirstDeployingAgent(envId);
-        long parallelThreshold = getFinalMaxParallelCount(envBean, totalNonFirstDeployAgents);
+        long parallelThreshold = extracted(envBean, totalNonFirstDeployAgents);
```

Comment:
what's extracted ?

Reply:
This PR is already closed after our discussion.
But I extracted the method to be able to unit testing the calculation without the need of complex testing for this.
The name was unhappy.

Reply:
yeah, I realized that it's closed. Please ignore comments :)